README.md

```diff
@@ -38,7 +38,7 @@ Tracetest is a OpenTelemetry based tool that helps you develop and test your dis
 
 - verify the quality of your OpenTelemetry instrumentation and enforce standards.
 - Testing events that occur on 'the other side' of an async message queue, even though the original async call has returned earlier.
-- Assertions based on the timing of different steps in your process.
+- Assertions based on the timing of different steps in your process, such as specifying maximum return time for a sql query or an external http call.
```

Comment:
Assertions based on the timing of different steps in your process, such as specifying maximum return time for a SQL query or an external HTTP call.